src/lib/middleware/locals.coffee

```diff
@@ -50,5 +50,8 @@ module.exports = (req, res, next) ->
     (ua.match(/BB10/i)) ||
     (ua.match(/BlackBerry/i))
   )
+  res.locals.sd.IS_IPAD = Boolean(
+    (ua.match(/iPad/i) && ua.match(/Mobile/i))
+  )
```

Comment:
We should probably address Android and other varieties of tablets -- https://developers.whatismybrowser.com/useragents/explore/hardware_type_specific/tablet/

Reply:
Good catch! I think the additional cases we will want to cover are:

```coffeescript
# specifically targets Vivo
(ua.match(/vivo/i) && ua.match(/Mobile/i)) ||
# targets android devices that are not mobile
(ua.match(/Android/i) && ua.match(/Mobile/i))
```

The one edge case I can think of is China MID Tablet because it is Android and Mobile.

Reply:
Also perhaps I should rename this constant to be `IS_TABLET`.